sasmodels/models/be_polyelectrolyte.py

```diff
@@ -84,11 +84,13 @@
 References
 ----------
 
+For further details, see [#Joanny1990]_, [#Moussaid1993]_, [#Raphael1990]_.
+
 .. [#Borue] V Y Borue, I Y Erukhimovich, *Macromolecules*, 21 (1988) 3240
-.. [#] J F Joanny, L Leibler, *Journal de Physique*, 51 (1990) 545
-.. [#] A Moussaid, F Schosseler, J P Munch, S Candau, *J. Journal de Physique
+.. [#Joanny1990] J F Joanny, L Leibler, *Journal de Physique*, 51 (1990) 545
+.. [#Moussaid1993] A Moussaid, F Schosseler, J P Munch, S Candau, *J. Journal de Physique
    II France*, 3 (1993) 573
-.. [#] E Raphael, J F Joanny, *Europhysics Letters*, 11 (1990) 179
+.. [#Raphael1990] E Raphael, J F Joanny, *Europhysics Letters*, 11 (1990) 179
```

Comment:
> Having one style for lists when the names are referenced in the text and another when they are not is less than ideal. It would be better to provide users with a canonical template they can follow.

Agreed, but a pile of very boring "See [1] [2] [3]." sentences that add nothing is also quite ugly.

> I'm guessing you added the "further details" sentence just so that all four sources would be referenced? 

Correct, see #378.

> Is no way to tell sphinx to ignore unused references?

None known. The combination of a "reading list" and a "references list" is quite an unusual construction; I don't know many bits of software that nicely handle that, particularly with a numbered reference style rather than author-date.